src/modules/checkLocal.js

```diff
@@ -50,14 +50,21 @@ const checkLocal = () => {
       });
 
       trash.addEventListener('click', () => {
-        localStorage.setItem('tasks', JSON.stringify(tasks));
+        // localStorage.setItem('tasks', JSON.stringify(tasks));
         li.remove();
         tasks.splice(tasks[i].index, 1);
         for (let i = 0; i < tasks.length; i += 1) {
           tasks[i].index = i;
         }
         localStorage.setItem('tasks', JSON.stringify(tasks));
       });
+
+      input.addEventListener('click', () => {
+        p.classList.toggle('lineThrough');
+        li.classList.toggle('completed');
+        tasks[tasks[i].index].completed = true;
+        localStorage.setItem('tasks', JSON.stringify(tasks));
+      });
```

Comment:
- [ ] The event listener is only changing the completed to true. If I untick the checkbox, the task's status is not changing to false. Please see the test demo of your app to understand what am trying to point out here.

Demo |
:-------:|
![a1](https://user-images.githubusercontent.com/39523312/171905493-6c5615cc-e49e-4efb-98a2-19ed7a267bfc.gif)

I will suggest working on making the checkbox update the task from false to true and true to false also.